### ISSUE TITLE
exception compiling on Mint 20

### ISSUE THREAD

**Issue by ax1ine** [fanoframework/fano-cli#9]
Trying to compile fano-cli on Mint 20 x64. After running './build.cfg' I get an exception. Here's the log:
`Free Pascal Compiler version 3.0.4 [2018/10/29] for x86_64`
`Copyright (c) 1993-2017 by Florian Klaempfl and others`
`Target OS: Linux for x86-64`
`Compiling src/fanocli.pas`
`Compiling ./src/Tasks/Implementations/Text2Inc/Text2IncTaskImpl.pas`
`Text2IncTaskImpl.pas(66,41) Error: Wrong number of parameters specified for call to "Create"`
`Text2IncTaskImpl.pas(66,41) Fatal: There were 1 errors compiling module, stopping`
`Fatal: Compilation aborted`
`Error: /usr/bin/ppcx64 returned an error exitcode`

is it something in the master branch?

**Comment by zamronypj** [comment]
@ax1ine  Thank you for bug report. This issue due to `TStringStream.create()' method signature change between Free Pascal 3.0.4 and Free Pascal 3.2.0 release. In Free Pascal 3.2.0 

```
constructor TStringStream.Create(const AString: string = ''); overload;
```
This issue cannot be reproduced in Free Pascal 3.2.0. I fixed this issue in 9754d59456  Please pull `master` branch and rebuild.

**Comment by zamronypj** [comment]
@ax1ine  Fixed 9754d59. Closing this one. In case you found bug, just create new issue. Thank you

**Comment by ax1ine** [comment]
thanks for the quick patch! i've just successfully compiled the fano-cli app :)
